__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const request = require('supertest')
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+
+beforeEach(() => {
+    return seed(testData)
+    });
+
+  afterAll(() => {
+    db.end()
+  });
+
+describe("GET API/topics", () => {
+    test("Should return an array", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeInstanceOf(Array)
```

Comment:
Technically as we have used a map later on, this means the response must be an array meaning we don't need to explicitly test for it here.